iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/AbstractTransformerXml.java

```diff
@@ -246,4 +250,19 @@ private static void readSteps(NetworkXmlReaderContext context, StepConsumer cons
         double rho = XmlUtil.readDoubleAttribute(context.getReader(), "rho");
         consumer.accept(r, x, g, b, rho);
     }
+
+    protected static void readRatedS(String name, NetworkXmlReaderContext context, RatedSConsumer consumer) {
+        double ratedS = XmlUtil.readOptionalDoubleAttribute(context.getReader(), name);
+        consumer.accept(ratedS);
```

Comment:
Rather than using a custom interface, you can use `DoubleConsumer` here and delete `RatedSConsumer`